src/plugins/NEMD/MettDeamon.cpp

```diff
@@ -1525,44 +1497,16 @@ void Reservoir::sortParticlesToBins(DomainDecompBase* domainDecomp, ParticleCont
 				_binQueue->enque(bin);
 			}
 			break;
-	}
-}
-
-void Reservoir::readFromMemory(DomainDecompBase* domainDecomp, ParticleContainer* particleContainer)
```

Comment:
So that feature was dropped completely? Why?

Reply:
This read method was neither ever used nor was it documented so it can be used in future. It furthermore made the code more complicated and error-prone.